app/controllers/stafftools_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StafftoolsController < ApplicationController
```

Comment:
I might have missed it elsewhere, but shouldn't this have some before-filter checking that the user is staff?